src/Containers/SessionChoice/index.tsx

```diff
@@ -59,21 +59,26 @@ const SessionChoice = () => {
       <CenteredContainer>
         <Row gutter={[0, 48]}>
           <Col span={12} offset={6}>
+            <Row justify='center'>
+              <Col>
+                <StyledImage src='/assets/logo.png' alt='logo' />
+              </Col>
+            </Row>
             <Row gutter={[0, 30]}>
               <Col span={24}>
                 <Tags>Nickname</Tags>
-                <Input maxLength={12} onChange={(e) => setNickname(e.target.value)}></Input>
+                <StyledInput maxLength={12} onChange={(e) => setNickname(e.target.value)}></StyledInput>
```

Comment:
On the backend we can handle nickname up to 15 characters

```suggestion
                <StyledInput maxLength={15} onChange={(e) => setNickname(e.target.value)} />
```